### PR TITLE
Contact Support from inside editor

### PR DESCRIPTION
**Pull request by twstokes**
This PR enables developing the feature to contact support from inside the editor for WPiOS and WPAndroid.

To test: Follow instructions in the [Gutenberg PR](https://github.com/WordPress/gutenberg/pull/34890).

## Related PRs

- https://github.com/WordPress/gutenberg/pull/34890
- https://github.com/wordpress-mobile/WordPress-iOS/pull/17175
- https://github.com/wordpress-mobile/WordPress-Android/pull/15328

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered if this change warrants user-facing release notes [more info](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/Release-notes.md) and have added them to [RELEASE-NOTES.txt](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/RELEASE-NOTES.txt) if necessary.

### CODE REVIEW

File: RELEASE-NOTES.txt
Comment (hypest) [inline RELEASE-NOTES.txt]: 👋 Tanner, what do you think about compacting the release note here? The total available space is not great in the app stores, and also can be longer after translating to other languages.

One proposal can be: `[**] Users can now contact support from inside the block editor screen. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3975]`

Comment (twstokes) [reply RELEASE-NOTES.txt]: Sure @hypest, glad to.

**Review by hypest — APPROVED** [review]
LGTM! I'm "approving" this PR to make it clear and yeah, besides updating the Gutenberg hash we'll need to revert the submodule pointer after we get the Gutenberg PR merged.

File: bundle/ios/GutenbergNativeTranslations.swift
Comment (twstokes) [inline bundle/ios/GutenbergNativeTranslations.swift]: @hypest can you help me understand why these changes occurred? Thank you.

Comment (hypest) [reply bundle/ios/GutenbergNativeTranslations.swift]: Aha, not exactly sure myself. @fluiddot (or @ceyhun) maybe you have some idea?

Comment (fluiddot) [reply bundle/ios/GutenbergNativeTranslations.swift]: The localizations string files are updated (via the `genstrings` command) when running the `npm run bundle` command:
https://github.com/wordpress-mobile/gutenberg-mobile/blob/3c906d04e6b0acf0b652ccaf082928e773c6145c/package.json#L66

The reason why this string got removed is related to an issue when parsing the files that contain that string:
- [gutenberg/packages/components/src/font-size-picker/index.native.js](https://github.com/WordPress/gutenberg/blob/039d087a127d59b524e41bb2104b9bbfe05750f2/packages/components/src/font-size-picker/index.native.js#L77)
- [packages/components/src/mobile/bottom-sheet-select-control/index.native.js](https://github.com/WordPress/gutenberg/blob/039d087a127d59b524e41bb2104b9bbfe05750f2/packages/components/src/mobile/bottom-sheet-select-control/index.native.js#L63)

### `font-size-picker`
**Command to test parsing:** `npm run wp -- i18n make-pot gutenberg/packages/components/src/font-size-picker --ignore-domain --debug test.pot`
**Error:** `Could not parse file index.native.js: Unexpected: > (line 86, column 4) (0.199s)`
**Line that failed:** https://github.com/WordPress/gutenberg/blob/039d087a127d59b524e41bb2104b9bbfe05750f2/packages/components/src/font-size-picker/index.native.js#L86

### `bottom-sheet-select-control`
**Command to test parsing:** `npm run wp -- i18n make-pot gutenberg/packages/components/src/mobile/bottom-sheet-select-control --ignore-domain --debug test.pot`
**Error:** `Could not parse file index.native.js: Unexpected: > (line 72, column 4) (0.209s)`
**Line that failed:** https://github.com/WordPress/gutenberg/blob/039d087a127d59b524e41bb2104b9bbfe05750f2/packages/components/src/mobile/bottom-sheet-select-control/index.native.js#L72

As you can see on both cases failed due to the use of short syntax of the fragment. This parsing issue is already fixed ([reference](https://github.com/wp-cli/i18n-command/issues/281)) on the WP-CLI command we use for this but until a new version of WP-CLI is released we'll, unfortunately, keep experiencing it 😞 .

In this case, I'd recommend reverting the change since the string is required and should be translated.

Comment (hypest) [reply bundle/ios/GutenbergNativeTranslations.swift]: Appreciate the investigation and the suggestion @fluiddot, thanks a mil!

Tangential but, maybe we talked about this in the past, could we perhaps patch-package WP-CLI until the fix is released?

Comment (fluiddot) [reply bundle/ios/GutenbergNativeTranslations.swift]: > Tangential but, maybe we talked about this in the past, could we perhaps patch-package WP-CLI until the fix is released?

Good point, I thought that by now a new WP-CLI version would have been released but since it's taking longer than expected, probably this approach should be considered 👍 .

Comment (twstokes) [reply bundle/ios/GutenbergNativeTranslations.swift]: Thank you @hypest and @fluiddot. These two files have been reverted and squashed into 961edcdbb2dd6b1549d97f8155f3464f2df85abf.

File: RELEASE-NOTES.txt
Comment (hypest) [reply RELEASE-NOTES.txt]: Addressed with https://github.com/wordpress-mobile/gutenberg-mobile/pull/3975/commits/827cae9d1a13aab53ec93e54c51ffdd9c7e761a5.

File: bundle/ios/GutenbergNativeTranslations.swift
Comment (hypest) [reply bundle/ios/GutenbergNativeTranslations.swift]: Looks good Tanner! Marking this thread as resolved 👍

**Comment by hypest** [comment]
👋 @twstokes , looks like all feedback has been addressed in this PR and the WP app PRs also got a green light from reviewers so, wdyt about resuming the merge domino? First step in that direction would be to update this PR to point to Gutenberg's PR and the merged (or later) commit. Then we'll smoke test the main app PRs to avoid having something obvious regressed and merge the gb-mobile PR. We'll need to tag the merged gb-mobile state with an `-alpha` tag to update in the main apps. Let me know if you need me to elaborate on any of that, thanks!

Edit: sorry, upon closer look, there's some [pending (design) review on the Gutenberg PR](https://github.com/WordPress/gutenberg/pull/34890#issuecomment-926271783) so, let's hold of until @kyleaparker gets the chance to have another pass.

**Comment by twstokes** [comment]
> 👋 @twstokes , looks like all feedback has been addressed in this PR and the WP app PRs also got a green light from reviewers so, wdyt about resuming the merge domino? First step in that direction would be to update this PR to point to Gutenberg's PR and the merged (or later) commit. Then we'll smoke test the main app PRs to avoid having something obvious regressed and merge the gb-mobile PR. We'll need to tag the merged gb-mobile state with an `-alpha` tag to update in the main apps. Let me know if you need me to elaborate on any of that, thanks!

Thanks @hypest! I believe all of that is clear to me - I appreciate the instructions!

**Pull request merged**
